README.md

```diff
@@ -43,6 +43,8 @@ Check out our [Next.js deployment documentation](https://nextjs.org/docs/deploym
 
 - [Neka Blair](https://github.com/istoga)
 - [Ojoachele Onuh](https://github.com/Achele)
+- Chibuzo Idia
```

Comment:
@Domicaren1 Can you add your GitHub as well

Reply:
Done